tests/audit/test_audit.py

```
@@ -499,7 +500,8 @@ def test_successful_contract_audit_request_dockerhub_fail_multiple_analyzers(sel
         # NOTE: if the audit node later requires the stubbed fields, this will have to change a bit
         self.__send_done_message(self.__REQUEST_ID)
         self.__assert_audit_request_report(self.__REQUEST_ID,
-                                           report_file_path="reports/DockerhubFailAllAnalyzers.json")
+                                           report_file_path="reports/DockerhubFailAllAnalyzers.json",
+                                           ignore_error=True)
```

So right now we're ignoring the error in `test_successful_contract_audit_request_dockerhub_fail_multiple_analyzers`, the rationale being that this has been tested in the smaller unit tests?

I was thinking whether we should also do something like 
`self.assertTrue(msg in report['errors'][0])` here as well, so that we still check the "error" in the dockerhub failure test. what do you think?

---

I was thinking the same thing, first check if error is related to pull access denied. If yes, then ignore_error=True else ignore_error=False.

Will that suffice?